docs/main/projects.md

```diff
@@ -11,7 +11,7 @@ last_updated: Sun Sep 06 14:28:14 2020 UTC
 - [Chrome Extension: Pi-Hole List Editor](https://chrome.google.com/webstore/detail/pi-hole-list-editor/hlnoeoejkllgkjbnnnhfolapllcnaglh) ([Source Code](https://github.com/packtloss/pihole-extension))
 - [Splunk: Pi-hole Visualiser](https://splunkbase.splunk.com/app/3023/)
 - [Adblocking with Pi-hole and Ubuntu 14.04 on VirtualBox](https://hbalagtas.blogspot.com/2016/02/adblocking-with-pi-hole-and-ubuntu-1404.html)
-- [Pi-hole stats in your Mac's menu bar](https://getbitbar.com/plugins/Network/pi-hole.1m.py)
+- [Pi-hole stats in your Mac's menu bar](https://github.com/matryer/bitbar-plugins/blob/master/Network/pi-hole.1m.py)

```

Comment:
Not sure about this; I went with linking to the py file directly.

The current redirect is: `https://getbitbar.com/plugins/Network/pi-hole.1m.py` -> `https://github.com/matryer/bitbar-plugins?from=getbitbar.com&dest=`

Reply:
It hasn't been updated in 3 years, go ahead and delete the whole line.

Reply:
Dan is right, we don't even know if this script is still working with the changes we've had in Pi-hole v5.0